src/pages/Popup/scripts/alarm.js

```diff
@@ -114,18 +114,41 @@ async function AddAlarm_click(course, dates) {
 }
 
 async function AddCustomAlarm(alarm) {
+  var resp = await new Promise((resolve) => chrome.storage.sync.get(resolve));
```

Comment:
is it working without passing `null` parameter

Reply:
yes